src/main/java/com/esri/samples/raster/raster_rendering_rule/README.md

```diff
@@ -0,0 +1,39 @@
+<h1>Raster Rendering Rule</h1>
+
+<p>Display a raster on a map and apply different rendering rules to that raster.</p>
+
+<p><img src="RasterRenderingRule.png"/></p>
+
+<h2>Use case</h1>
+
+<p>Image Service Rasters may provide several rendering rules options. When it is necessary to switch between rendering rules in order to display the desired data, the interface for the selection and application of the required rule can be used.</p>
```

Comment:
> the interface for the selection and application of the required rule can be used

I don't know what this means. Please reach out to the team for a real-world use case.

Reply:
I'd go for something like this. When you make a PR for this in the Common Samples repo, the team may have other suggestions!

"A single raster image can be rendered in a number of different ways, including representation of slope aspect, hillshade and shaded relief. Applying these different renderers to the same raster allows for a powerful visual analysis of the data. For example, a geologist could interrogate the raster image using all available renderers to map subtle geological features on a landscape not apparent in a regular aerial photograph."

Reply:
Very wel put, thanks!